ray-operator/main.go

```diff
@@ -176,14 +179,14 @@ func main() {
 	exitOnError(mgr.Start(ctrl.SetupSignalHandler()), "problem running manager")
 }
 
-func cacheSelectors() (cache.SelectorsByObject, error) {
+func cacheSelectors() (map[client.Object]cache.ByObject, error) {
 	label, err := labels.NewRequirement(common.KubernetesCreatedByLabelKey, selection.Equals, []string{common.ComponentName})
 	if err != nil {
 		return nil, err
 	}
 	selector := labels.NewSelector().Add(*label)
 
-	return cache.SelectorsByObject{
+	return map[client.Object]cache.ByObject{
 		&batchv1.Job{}: {Label: selector},
```

Comment:
A dumb question: Does it mean that the informer only caches `batchv1.Job`?

Reply:
There is no dumb question :)

Anything that's not specified here falls through the defaults. Excerpted from the `ByObject` documentation:
```
// The defaulting follows the following precedence order:
// 1. ByObject
// 2. DefaultNamespaces[namespace]
// 3. Default*
```

Some examples can be found at https://github.com/kubernetes-sigs/controller-runtime/blob/2154ffbc22e26ffd8c3b713927f0df2fa40841f2/designs/cache_options.md, as well as some details at kubernetes-sigs/controller-runtime#2421.

Reply:
Got it. This is very helpful!